src/slic3r/GUI/GUI_App.cpp

```diff
@@ -1937,9 +1937,6 @@ void GUI_App::init_app_config()
         if (!boost::filesystem::exists(data_dir_path)){
             boost::filesystem::create_directory(data_dir_path);
         }
-
```

Comment:
We need this otherwise some wired tuk log files will be generated in application folder if Bambu network plugin is used

Reply:
That's a problem then.

libwebkit2gtk refers to webkit binaries located at ```/usr/lib/x86_64-linux/gnu/webkit2gtk-4.0/``` using absolute paths. I modify ```libwebkit2gtk-4.0.so``` replacing absolute path with relative one ```././/lib/x86_64-linux-gnu/webkit2gtk-4.0``` just like recommended by AppImage documentation so bundled webkit would be used instead on system-supplied at ```/usr/...```

In theory it should be possible to intercept system calls using LD_PRELOAD trick, but I couldn't get it working using [appimage-builder](https://appimage-builder.readthedocs.io/en/latest/) which is using hooks to manipulate file paths.

I'll look what could be done for this.